app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<h2>Sign up</h2>
+
+<%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= f.error_notification %>
+
+  <div class="form-inputs">
+    <%= f.input :email,
+                required: true,
+                autofocus: true,
+                input_html: { autocomplete: "email" }%>
+    <%= f.input :password,
+                required: true,
+                hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length),
+                input_html: { autocomplete: "new-password" } %>
+    <%= f.input :password_confirmation,
+                required: true,
+                input_html: { autocomplete: "new-password" } %>
+  </div>
+
+  <div class="form-actions">
+    <%= f.button :submit, "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] I cannot sign up a user. You do not have `name` field on the form so it causes validation errors. Kindly the following code at the top of your form.

  ```
   <%= f.input :name,
     required: true,
     autofocus: true,
     input_html: { autocomplete: "name" }%>
  ```

- [ ] Kindly center the sign up form also.

  ![image](https://github.com/lucy-sees/Blog/assets/54848363/1bc2e75d-03cc-429c-8d7d-e361c1aec472)